cross/ffmpeg/Makefile

```diff
@@ -64,23 +82,31 @@ CONFIGURE_ARGS += --enable-libopus
 DEPENDS += cross/soxr
 CONFIGURE_ARGS += --enable-libsoxr
 
+ifneq ($(findstring $(ARCH),$(OLD_PPC_ARCHS)),$(ARCH))
 DEPENDS += cross/x264
 CONFIGURE_ARGS += --enable-libx264
+endif
 
+ifneq ($(findstring $(ARCH),$(OLD_PPC_ARCHS)),$(ARCH))
 DEPENDS += cross/x265
 CONFIGURE_ARGS += --enable-libx265
+endif
 
 DEPENDS += cross/opencore-amr
 CONFIGURE_ARGS += --enable-libopencore-amrnb --enable-libopencore-amrwb
 
+ifneq ($(findstring $(ARCH),$(OLD_PPC_ARCHS)),$(ARCH))
 DEPENDS += cross/libvpx
 CONFIGURE_ARGS += --enable-libvpx
+endif
 
 DEPENDS += cross/libzmq
 CONFIGURE_ARGS += --enable-libzmq
 
+ifneq ($(findstring $(ARCH),$(OLD_PPC_ARCHS)),$(ARCH))
 DEPENDS += cross/shine
 CONFIGURE_ARGS += --enable-libshine
+endif
```

Comment:
I'd rather have a single `ifneq` to deal with instead of multiple repeat of it.

Reply:
So you don't mind when I change the order to have all in one group for non OLD_PPC_ARCHS?

Reply:
Nah, I don't mind.  I prefer a single group to manage that one exception.

Reply:
done